src/modules/0.0.22/aws_s3/rpcs/s3_clean_bucket.ts

```diff
@@ -0,0 +1,75 @@
+import { S3, _Object } from '@aws-sdk/client-s3';
+
+import { AwsS3Module } from '..';
+import { AWS, crudBuilder2, crudBuilderFormat } from '../../../../services/aws_macros';
+import { Context, RpcBase, RpcResponseObject } from '../../../interfaces';
+
+export class S3CleanBucketRpc extends RpcBase {
+  module: AwsS3Module;
+  outputTable = {
+    bucket: 'varchar',
+    status: 'varchar',
+    response_message: 'varchar',
+  } as const;
+
+  getBucketObjects = crudBuilderFormat<S3, 'listObjectsV2', _Object[]>(
+    'listObjectsV2',
+    bucketName => ({ Bucket: bucketName }),
+    res => res?.Contents ?? [],
+  );
+  deleteBucketObject = crudBuilder2<S3, 'deleteObject'>('deleteObject', (bucketName, key) => ({
+    Bucket: bucketName,
+    Key: key,
+  }));
+
+  call = async (
+    dbId: string,
+    dbUser: string,
```

Comment:
nit: if they are not in use
```suggestion
    _dbId: string,
    _dbUser: string,
```